cli/service.js

```diff
@@ -0,0 +1,118 @@
+import asciify from "asciify-image";
+import inquirer from "inquirer";
+import fs from "fs";
+import chalk from "chalk";
+import ItemManager from "./item-manager.js";
+import PokemonClient from "./pokemon-client.js";
+
+const POKEMON_FILE_NAME = "pokemon-names.json";
+const TASKS_FILE_NAME = "tasks.json";
+const tasksManager = new ItemManager(render);
+const pokemonClient = new PokemonClient();
+let allPokemonNames = [];
+
+export async function init() {
+   tasksFileInit();
+   pokemonFileInit();
+
+   return tasksManager;
+}
+
+function tasksFileInit() {
+   if (!fs.existsSync(TASKS_FILE_NAME)) {
+      fs.writeFileSync(TASKS_FILE_NAME, JSON.stringify([]));
+   } else {
+      const fileData = fs.readFileSync(TASKS_FILE_NAME);
+      let fileTasks = JSON.parse(fileData);
+      fileTasks.forEach((task) => {
+         tasksManager.add(task);
+      });
+   }
+}
+
+async function pokemonFileInit() {
+   if (!fs.existsSync(POKEMON_FILE_NAME)) {
+      allPokemonNames = await getAllPokemonNames();
+      let data = JSON.stringify(allPokemonNames);
+      fs.writeFileSync(POKEMON_FILE_NAME, data);
+   } else {
+      allPokemonNames = JSON.parse(fs.readFileSync(POKEMON_FILE_NAME));
+   }
+}
+
+async function getAllPokemonNames() {
+   const pokemonNames = await pokemonClient.getAllPokemonNames();
+   return JSON.stringify(pokemonNames);
+}
+
+function render() {
+   let data = JSON.stringify(tasksManager.items);
+   fs.writeFileSync(TASKS_FILE_NAME, data);
+}
+
+export async function chooseTaskToDelete() {
+   const answer = await inquirer.prompt({
+      name: "delete",
+      type: "list",
+      message: "Which task would you like to delete?",
+      choices: tasksManager.items,
+   });
+   return tasksManager.items.indexOf(answer.delete);
+}
+
+export async function addTask(task) {
+   if (
+      task
```

Comment:
The input parsing should be made in the cli class. only the logic of the adding task is relevant here.